stonesoup/feeder/tests/test_filter.py

```diff
@@ -28,3 +34,83 @@ def test_metadata_reducer(detector):
         assert all(time == detection.timestamp for detection in detections)
 
     assert multi_none
+
+
+def test_metadata_value_filter(detector):
+    feeder = MetadataValueFilter(detector,
+                                 metadata_field="score",
+                                 operator=lambda x: operator.ge(x, 0.1))
```

Comment:
Does the following work?
```python
lambda x: x < 0.1
```

May be easier than trying to explain use of `operator` module in your documentation string.

Reply:
I have completely removed any mention and use of the `operator` module, relating to the newly introduced class. I agree that basing the explanation around it can be confusing to new users.

The `operator` module was used more heavily in the initial draft of the class, which I ended up modifying to add more flexibility.